resources/exes/thumb-patch-space/config.json

```diff
@@ -17,25 +17,14 @@
                      output = x * three;",
      "patch-point" : "0x103ee",
      "patch-size" : 4,
-     "patch-vars": [
+     "patch-vars" : [
        {"name": "input",
-	      "at-entry": {
-          "stored-in": "register",
-	        "register": "R3"
-        },
-        "at-exit": {
-          "stored-in": "register",
-	        "register": "R3"
-        }},
+        "at-entry": "R0"
```

Comment:
Why is this one changing from R3 to R0?  I'm surprised the test even passes, since I don't see a corresponding change to the binary.

Reply:
```
   103e6:       230a            movs    r3, #10
   103e8:       603b            str     r3, [r7, #0]
   103ea:       683b            ldr     r3, [r7, #0]
   103ec:       0018            movs    r0, r3
   103ee:       f7ff ffeb       bl      103c8 <g> ; <--- this is the patch point
   103f2:       0003            movs    r3, r0
   103f4:       607b            str     r3, [r7, #4]
   103f6:       687b            ldr     r3, [r7, #4]
   103f8:       0018            movs    r0, r3
   ```

Technically, `input` is in both `r0` and `r3`, at the entry point, but we don't care what register it's in at exit. Meanwhile, we want `output` to be in `r0` when our patch jumps back to `103f2`.

Reply:
Got it - thanks!